enterprise/app/controllers/api/v1/accounts/applied_slas_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Api::V1::Accounts::AppliedSlasController < Api::V1::Accounts::EnterpriseAccountsController
+  include Sift
+  include DateRangeHelper
+
+  RESULTS_PER_PAGE = 25
+
+  before_action :set_applied_slas, only: [:index, :metrics]
+  before_action :set_current_page, only: [:index]
+  before_action :set_current_page_appiled_slas, only: [:index]
+  before_action :check_admin_authorization?
+
+  sort_on :created_at, type: :datetime
+
+  def index; end
+
+  def metrics
+    @total_applied_slas = total_applied_slas
+    @number_of_sla_breaches = number_of_sla_breaches
+    @hit_rate = hit_rate
+  end
```

Comment:
We need not assign them as instance variables, we can use the methods directly in `metrics.json.jbuilder`

Either way since, the methods do the assignment themselves, we need to assign it again